components/org.wso2.carbon.analytics.msf4j.interceptor.common/pom.xml

```diff
@@ -0,0 +1,89 @@
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wso2.carbon.analytics</groupId>
+        <artifactId>org.wso2.carbon.analytics.parent</artifactId>
+        <version>2.0.196-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.carbon.analytics.msf4j.interceptor.common</artifactId>
+    <packaging>bundle</packaging>
+
+    <name>WSO2 Stream Processor - Interceptors</name>
+    <url>http://maven.apache.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.config</groupId>
+            <artifactId>org.wso2.carbon.config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.msf4j</groupId>
+            <artifactId>msf4j-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.analytics-common</groupId>
+            <artifactId>org.wso2.carbon.analytics.idp.client</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <basedir>${project.build.outputDirectory}/org/wso2/carbon/analytics/msf4j/interceptor/common
+                    </basedir>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
+        <import.package>
+            org.wso2.carbon.config.*;version="${carbon.config.version.range}",
+            org.wso2.carbon.analytics.idp.client.*,
+            org.wso2.carbon.kernel.*,
+            org.osgi.framework.*,
+            org.wso2.msf4j.*,
+            javax.ws.rs.*,
+            javax.xml.bind*,
+            org.slf4j.*,
+            io.swagger.annotations.*,
+            org.wso2.carbon.messaging.*
+        </import.package>
+        <private.package>
+            org.wso2.carbon.analytics.msf4j.interceptor.common.internal,
+        </private.package>
+        <export.package>
+            !org.wso2.carbon.analytics.msf4j.interceptor.common.internal,
+            org.wso2.carbon.analytics.msf4j.interceptor.common.*
```

Comment:
export with the version, see https://github.com/wso2/carbon-ui-server/blob/v0.16.0/components/org.wso2.carbon.uis/pom.xml#L175-L178

Reply:
Fixed in 904c750